.azure-pipelines/win32-conda-CI.yml

```diff
@@ -36,12 +36,10 @@ jobs:
       python -m pip install --upgrade pip
       pip install -r requirements.txt
       pip install -r requirements-dev.txt
-      test '$(python.version)' != '2.7' && pip install onnxruntime
```

Comment:
Why are you removing this from here? This is present in linux-conda-CI.yml.

Reply:
No way to python 2.7 in Windows since there is no available onnx package for that.